lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -47,11 +47,14 @@
 // styles together, so that card becomes more of
 // a styled container.
 .beta-card {
-  background-color: $white;
```

Comment:
I'm also not 100% convinced that we should be removing this from the card pattern, since it seems like something we'd want nearly everywhere else. Are you sure having a background color on the container is causing a problem here?